SS2_Dockerfiles/Picard/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM java:openjdk-8-jre
+ENV version 2.10.10
+WORKDIR /usr/picard
+ADD https://github.com/broadinstitute/picard/releases/download/${version}/picard.jar  
```

Comment:
Should we be consistent between dockerfiles so that they all have the 
`MAINTAINER` and `LABEL` description tags?